docs/src/pages/components/masonry/BasicMasonry.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function BasicMasonry(): JSX.Element {
```

Comment:
We usually omit it

```suggestion
export default function BasicMasonry() {
```

x the other cases